src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Physical Cluster Replication Technical Overview
+summary: Understand the technical details of CockroachDB physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication uses two clusters with _cluster virtualization_ enabled to stream data from an active primary cluster to a passive standby cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Start the replication stream
+
+When you initiate physical cluster replication, it will replicate the application virtual cluster from the primary to the standby cluster. The replication stream sets up a series of jobs that complete the following work:
+
+1. The standby's system interface communicates to the primary cluster that a stream should start.
+1. The primary cluster chooses a timestamp at which to start the stream and lays down a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps), which will protect the entirety of the cluster's data from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until an initial scan is complete.
+1. The initial scan runs on the primary and backfills all data from the primary virtual cluster up to the starting timestamp of the replication stream.
+1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
+1. The standby cluster uses a PG connection to retrieve all of the primary cluster's node addresses. This generates a topology of the primary cluster. If there is a mismatch in nodes between the primary and standby, the replication job will share out the work between the standby's nodes.
```

Comment:
This and the preceeding step should go above the step "Initial scan runs on the primary" step. Assuming we've mentioned the SQL connection above, I would say:

1. The standby cluster retrieves a list of all nodes in the primary cluster. It uses this list to distribute work across all nodes in the standby cluster.

Reply:
Done!